### PR TITLE
add English quick start && polish some docs of chinese version

### PR DESCRIPTION
**Pull request by oldLady344**

### CODE REVIEW

File: docs/en/quick-start/st-ug-create-schema.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Create a schema
+
+To bulk import data into a Nebula Graph database, you must have a schema. You can create a schema on **Console** or **Schema** of Studio.
+
+> **NOTE**: You can use nebula-console to create a schema. For more information, see [Deploy Nebula Graph with Docker Compose
+](https://github.com/vesoft-inc/nebula-docker-compose/blob/master/README.md "Click to go to GitHub website") and [Get started with Nebula Graph](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/1.get-started/ "Click to go to Nebula Graph website").
```
Comment (randomJoe211) [inline docs/en/quick-start/st-ug-create-schema.md:6]: Use links closely related to the content and can be used by readers directly. 
Maybe attach the GitHub page of nebula-console and the section describing how to create schema?

File: docs/en/quick-start/st-ug-import-data.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Import data
+
+When you have a schema and CSV files of data, you can use the **Import** page to bulk import vertex and edge data into a Nebula Graph database for graph exploration and data analysis.
```
Comment (randomJoe211) [inline docs/en/quick-start/st-ug-import-data.md:3]: "When you have a schema and CSV files of data" seems to be grammatically incorrect, or at least it reads weird. Please consider reorganizing it.

File: docs/en/quick-start/st-ug-create-schema.md (hunk 1 above)
Comment (oldLady344) [reply docs/en/quick-start/st-ug-create-schema.md:6]: > Use links closely related to the content and can be used by readers directly.
> Maybe attach the GitHub page of nebula-console and the section describing how to create schema?

I want to direct readers to the "Deployment with Docker Compose" doc, not the nebula-console repo.

File: docs/en/quick-start/st-ug-create-schema.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Create a schema
+
+To bulk import data into a Nebula Graph database, you must have a schema. You can create a schema on **Console** or **Schema** of Studio.
```
Comment (whitewum) [inline docs/en/quick-start/st-ug-create-schema.md:3]: what is "Console" or "Schema". It's confusing. do you mean "Console tab or Schema tab on Studio"?

File: docs/en/quick-start/st-ug-create-schema.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   ```nGQL
+    CREATE TAG user (userId int); -- To create a tag named "user", with one property
+    CREATE TAG course (courseId int, courseName string); -- To create a tag named "course", with two properties
+    CREATE EDGE action (actionId int, duration double, label bool, feature0 double, feature1 double, feature2 double, feature3 double); -- To create an edge type named "action", with seven properties
+    ```
+
+If the preceding statements are executed successfully, the schema is created. You can run the statements as follows to view the schema.
+
+```nGQL
+SHOW TAGS; -- To list all the tags in the current graph space
```
Comment (whitewum) [inline docs/en/quick-start/st-ug-create-schema.md:45]: better to paste the results.

**Review by whitewum — APPROVED** [review]
better to submit multiple pull requests which contain 1 or 2 files.

**Review by whitewum — APPROVED** [review]

**Pull request merged**